lively.morphic/world-loading.js

```diff
@@ -82,7 +83,7 @@ export async function loadWorld (newWorld, oldWorld, options = {}) {
       window.history.pushState({}, 'lively.next', options.pathForBrowserHistory);
     }
 
-    newWorld.showsUserFlap = showUserFlap;
+    newWorld.showsUserFlap = showUserFlap; // will initialize Comment Browser
```

Comment:
But this was already triggered. Is there a race condition of some sorts?

Reply:
What do you mean this was already triggered?

Reply:
I meant the initialization of the Comment Browser.
The property at that point as already been set to that value. 
This therefore triggers the initialization of the comment browser just after the user widgets habe been loaded.
So this assignment does not really do anything as far as I can tell.

Reply:
This is the only line where `showUserFlap` is set. If you do not do that, loading a new world results in not having a top bar at all. We just added the comment which does not totally display what we wanted to say.
What we really wanted to say is, that here is the point what triggers to initialize the comment browser. If there is no top bar no comment browser gets initialized.

Reply:
Beats me, sorry I was confused by the git diff.